api/src/main/java/eu/coatrack/api/Proxy.java

```diff
@@ -55,6 +56,10 @@ public Proxy(String id) {
 
     private String configServerPassword;
 
+    private LocalDateTime lastCallTimeToAdmin;
+
+    private Long minutesPastSinceLastContact;
```

Comment:
This can be calculated from the last call time, so why store it in the database?

Reply:
To be easier and clean to read by thymeleaf, or I would need to add more on value on the Model and View to be sent to the Thymeleaf

Reply:
Options:
- Create Proxy Object

Reply:
Object that saves all the Gateway Health Monitor data is created and is working.
So from now on only the last time of the successful call to admin is saved on database.
With this, I consider this Done